shared/common-adapters/button.js.flow

```diff
@@ -7,11 +7,17 @@ export type Props = {
   label: ?string,
   style?: ?Object,
   primary?: ?bool,
+  secondary?: ?bool,
+  follow?: ?bool,
```

Comment:
I agree that these should be enums, because it doesn't make sense to be able to set danger: true, and secondary: true.